packages/mobile_app/lib/common/ambiance/common.dart

```diff
@@ -0,0 +1,188 @@
+export 'package:mutable_wrappers/mutable_wrappers.dart';
+export 'package:my_horoskope/app_global.dart';
+export 'package:my_horoskope/common/gradient_flatbutton.dart';
+export 'package:my_horoskope/common//gradient_border_button.dart';
+export 'package:base/int_datetime.dart';
+export 'package:flutter_svg/flutter_svg.dart';
+export 'package:my_horoskope/common//triangle_painter.dart';
+
+class AmbianceUserInputFields extends StatelessWidget {
+  final MutableString name;
+  final MutableString role;
+  final MutableString month;
+  final MutableString day;
+  final MutableString year;
+  final MutableInteger sex;
+
+  AmbianceUserInputFields({
+    Key key,
+    @required this.name,
+    @required this.role,
+    @required this.day,
+    @required this.month,
+    @required this.year,
+    @required this.sex,
+  }) : super(key: key);
+
+  final indexToSex = {
+    0: localeText.notSelectedSex.capitalize(),
+    1: localeText.male.capitalize(),
+    2: localeText.female.capitalize(),
+    3: localeText.other.capitalize(),
+  };
+
+  @override
+  Widget build(BuildContext context) {
+    final node = FocusScope.of(context);
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        // name
+        UserInfoField(
+          textObject: name,
+          maxLength: 100,
+          keyboardType: TextInputType.name,
+          capitalizationType: TextCapitalization.words,
+          hint: localeText.name.capitalize(),
+          onEditingComplete: () => node.nextFocus(),
+          validator: (String text) {
+            int min = 2;
+            if (text.isEmpty || text.length < min)
+              return "${localeText.atLeastXsymbolsNeeded} $min";
+            return null;
+          },
+        ),
+        // role
+        UserInfoField(
+          textObject: role,
+          maxLength: 100,
+          keyboardType: TextInputType.name,
+          capitalizationType: TextCapitalization.words,
+          hint: localeText.role.capitalize(),
+          onEditingComplete: () => node.nextFocus(),
+          validator: (String text) {
+            int min = 1;
+            if (text.isEmpty || text.length < min)
+              return "${localeText.atLeastXsymbolsNeeded} $min";
+            return null;
+          },
+        ),
+        Padding(
+          padding: const EdgeInsets.only(top: 16.0, left: 4.0),
+          child: Text(
+            localeText.birthdate.capitalize(),
+            style: AppTextStyle.fadeText,
+          ),
+        ),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.start,
+          children: <Widget>[
+            Expanded(
+              flex: 2,
+              child: Padding(
+                padding: const EdgeInsets.only(right: 16),
+                child: UserInfoField(
+                  textObject: day,
+                  maxLength: 2,
+                  keyboardType: TextInputType.datetime,
+                  hint: "dd",
```

Comment:
Shouldn't it be localized?

Reply:
No, we didn't localise it in settings/register screens too

Reply:
Follow up #110